packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -747,22 +748,34 @@ enum _CurrentSection {
 /// The byte order used is [Endian.host] throughout.
 class VectorGraphicsBuffer {
   /// Creates an interface for incrementally building a [ByteData] instance.
-  VectorGraphicsBuffer()
-      : _buffer = <int>[],
-        _isDone = false,
-        _eightBytes = ByteData(8) {
-    _eightBytesAsList = _eightBytes.buffer.asUint8List();
+  ///
```

Comment:
Uses Uint8List backing like we reverted to on the framework, with an initial capacity hint based on the svg size